devnet-sdk/devstack/presets/contracts.go

```diff
@@ -0,0 +1,10 @@
+package presets
+
+import "github.com/ethereum-optimism/optimism/devnet-sdk/devstack/sysgo"
+
+func contractPaths() sysgo.ContractPaths {
+	return sysgo.ContractPaths{
+		FoundryArtifacts: "../../../packages/contracts-bedrock/forge-artifacts",
+		SourceMap:        "../../../packages/contracts-bedrock",
+	}
```

Comment:
We need something to automatically resolve paths relative to the monorepo root, so that these type of things become reusable when called from different packages.

Reply:
Good call.  Added an iterative search to find the root based on the existence of `packages/contracts-bedrock`.